README.md

```diff
@@ -100,6 +100,37 @@ class Scanner(AbstractScanner):
 
 ```
 
+Here is an example for using the `malwarerepoclient` utility in Engine unit tests
+
+```python
+import asyncio
+import pytest
+import sys
+
+from malwarerepoclient.client import DummyMalwareRepoClient
```

Comment:
Does this import still works, after moving the code into `microengineutils` module?
```suggestion
from microengine_utils.malwarerepoclient.client import DummyMalwareRepoClient
```